.github/ISSUE_TEMPLATE/discussion.md

```diff
@@ -1,7 +1,18 @@
 ---
-name: I want to discuss or propose a change to React Native.
+name: 🚀 Discussion or Proposal
 about: You have an idea that could make React Native better, or you want to discuss some aspect of the framework.
+labels: "Type: Discussion"
 ---
 
 The React Native community has set up a separate repository to track discussions and proposals:
```

Comment:
Ahhh, this explains why people go there when they have questions for React Native.

Can we flip around the paragraph below and this one? Basically the first one should be discord, spectrum and the FB group.

Then below that we could say something like "If you'd like to discuss topics related to the future of React Native, please check out the discussions and proposals repo: …"

That may help reduce the wrong issue from being created there.